### PR TITLE
Adding Chinese version of README

### PR DESCRIPTION
**Pull request by 239573049**
## Motivation and Context

- This is about adding Chinese documentation.
- Addresses nanoframework/Home#1146.

## Types of changes

- [ ] Improvement (non-breaking change that improves a feature, code or algorithm)
- [ ] Bug fix (non-breaking change which fixes an issue with code or algorithm)
- [ ] New feature (non-breaking change which adds functionality to code)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Config and build (change in the configuration and build system, has no impact on code or features)
- [ ] Dependencies (update dependencies and changes associated, has no impact on code or features)
- [ ] Unit Tests (add new Unit Test(s) or improved existing one(s), has no impact on code or features)
- [x] Documentation (changes or updates in the documentation, has no impact on code or features)

## Checklist:

- [ ] My code follows the code style of this project (only if there are changes in source code).
- [ ] My changes require an update to the documentation (there are changes that require the docs website to be updated).
- [x] I have updated the documentation accordingly (the changes require an update on the docs in this repo).
- [ ] I have read the [CONTRIBUTING](https://github.com/nanoframework/.github/blob/master/CONTRIBUTING.md) document.
- [ ] I have tested everything locally and all new and existing tests passed (only if there are changes in source code).
- [ ] I have added new tests to cover my changes.

### CODE REVIEW

File: README.zh-cn.md [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+
+# 欢迎使用.net **nanoFramework** IoT.Device Library repository
+
+该存储库包含绑定，可以是传感器，小屏幕和任何您可以连接到纳米框架芯片的东西!
+
+大多数绑定都是从[.NET IoT repository](https://github.com/dotnet/iot/tree/main/src/devices)迁移过来的。并不是所有的绑定都对迁移到.net纳米框架有意义，所以迁移的工作已经放在了能够与.net nanoFramework 工作的设备上。也请注意，一些设备已经迁移，没有测试，所以它们主要包含问题。
+
+## 设备列表
+
+<devices>
```
Comment (Ellerbach) [inline README.zh-cn.md:16]: all this part is generated automatically. So you should adjust the code from the application generating this part.
Here: https://github.com/nanoframework/nanoFramework.IoT.Device/tree/develop/src/device-listing

So adjust the code to generate this section from the device listing code.

**Comment by Ellerbach** [comment]
@239573049 you can also use this tool to help in the translation: https://github.com/Ellerbach/docfx-companion-tools/tree/main/src/DocLanguageTranslator

So far, it's working on folder structure. It can be adjusted for naming convention. It uses Azure Cognitive Services for the translation (a key is required but free should be enough for what you need to do). It can help bootstrap the localization.

**Comment by 239573049** [comment]
> @239573049 你也可以使用这个工具来帮助翻译：https://github.com/Ellerbach/docfx-companion-tools/tree/main/src/DocLanguageTranslator
> 
> 到目前为止，它正在研究文件夹结构。可以针对命名约定对其进行调整。它使用 Azure 认知服务进行转换（密钥是必需的，但免费应该足以满足你需要执行的操作）。它可以帮助引导本地化。

That's not very good,

When I translate, I try to understand what he means to see if he can express it, and if the full machine translation doesn't work well

> @239573049 you can also use this tool to help in the translation: https://github.com/Ellerbach/docfx-companion-tools/tree/main/src/DocLanguageTranslator
> 
> So far, it's working on folder structure. It can be adjusted for naming convention. It uses Azure Cognitive Services for the translation (a key is required but free should be enough for what you need to do). It can help bootstrap the localization.

That's not very good,

When I translate, I try to understand what he means to see if he can express it, and if the full machine translation doesn't work well

**Comment by Ellerbach** [comment]
> When I translate, I try to understand what he means to see if he can express it, and if the full machine translation doesn't work well

You can give it a try, it's been proven to help in projects I've been involved. Still a human has to adjust after but it has the advantage of keep the structure, the links, all this.

**Comment by 239573049** [comment]
> > When I translate, I try to understand what he means to see if he can express it, and if the full machine translation doesn't work well
> 
> You can give it a try, it's been proven to help in projects I've been involved. Still a human has to adjust after but it has the advantage of keep the structure, the links, all this.
I was to use the program to translate after observing whether there are translation problems, so efficient

**Comment by josesimoes** [comment]
@239573049 any progress on this?

**Pull request closed without merging**